tock/tock/templates/hours/timecard_form.html

```diff
@@ -10,6 +10,26 @@
     </p>
   </div>
 </div>
+{% if float_data.tasks %}
+  <div class="usa-alert usa-alert-error">
+    <div class="usa-alert-body">
+      <p class="usa-alert-text">
+        <b>
+          For {{ float_data.metadata.float_period_start }} to
+          {{ float_data.metadata.float_period_end }} you were scheduled in Float
+          for:
+        </b>
+      </p>
+    <ul>
+      {% for i in float_data.tasks %}
+       <li>-  {{ i.hours_wk }} hours on {{ i.project_name }}
```

Comment:
Hmm, I'll have to take a look at this separately--since I'm working on USWDS now, I gotta make sure it's free of confusing paper cuts like these.

That said, I just noticed that Tock is using USWDS 0.12, so it might have been fixed in a recent revision.  Hmmmmmm!